src/test/JerryProtocolHandler.test.ts

```diff
@@ -54,28 +54,28 @@ suite('JerryProtocolHandler', () => {
 
     test('allows otherwise valid message to be too long', () => {
       delegate.onError.resetHistory();
-      const array = Uint8Array.from([0, 200, 4, 1, SP.JERRY_DEBUGGER_VERSION, 0]);
+      const array = Uint8Array.from([0, 1, SP.JERRY_DEBUGGER_VERSION, 0, 0, 0, 200, 4, 0]);
       handler.onConfiguration(array);
```

Comment:
Could you please define an array with 4 bytes for the JERRY_DEBUGGER_VERSION and concat it with the other argument bytes instead of this part solution? It occurs in several places in the test cases, please modify those too.